### PR TITLE
facial occupancy

### PR DESCRIPTION
**Pull request by yog27ray**

### CODE REVIEW

File: cam2/src/main/java/com/commonsware/cwac/cam2/helper/ImageHelper.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import com.google.android.gms.vision.Frame;
+import com.google.android.gms.vision.face.Face;
+import com.google.android.gms.vision.face.FaceDetector;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ImageHelper {
+  public boolean isFaceOccupancy(Context context, Bitmap bitmap, float percent) {
```
Comment (ramakrishna-git) [inline cam2/src/main/java/com/commonsware/cwac/cam2/helper/ImageHelper.java]: return an enum for following cases
1. no face found
2. more than face found
3. one face found but smaller than the required percentage
4. Face found with required percentage

Comment (ramakrishna-git) [inline cam2/src/main/java/com/commonsware/cwac/cam2/helper/ImageHelper.java]: rename isFaceOccupancy to isFacePresentWithMinimumOccupancy
rename percent to minimumOccupancyPercentage

File: cam2/src/main/java/com/commonsware/cwac/cam2/helper/ImageHelper.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+import com.google.android.gms.vision.Frame;
+import com.google.android.gms.vision.face.Face;
+import com.google.android.gms.vision.face.FaceDetector;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ImageHelper {
```
Comment (ramakrishna-git) [inline cam2/src/main/java/com/commonsware/cwac/cam2/helper/ImageHelper.java]: rename file to FaceOccupancyDetector

File: cam2/src/main/java/com/commonsware/cwac/cam2/helper/ImageHelper.java (hunk 1 above)
Comment (yog27ray) [reply cam2/src/main/java/com/commonsware/cwac/cam2/helper/ImageHelper.java]: done

Comment (yog27ray) [reply cam2/src/main/java/com/commonsware/cwac/cam2/helper/ImageHelper.java]: done

File: cam2/src/main/java/com/commonsware/cwac/cam2/helper/ImageHelper.java (hunk 2 above)
Comment (yog27ray) [reply cam2/src/main/java/com/commonsware/cwac/cam2/helper/ImageHelper.java]: done

**Pull request merged**